apps/web/src/views/RemoveLiquidity/RemoveLiquidityV3.tsx

```diff
@@ -183,22 +187,26 @@ function Remove({ tokenId }: { tokenId?: bigint }) {
 
     const publicClient = getViemClients({ chainId })
 
-    publicClient.estimateGas(txn).then((gas) => {
-      sendTransactionAsync({
```

Comment:
it's a bug on wagmi end. Someplace in error handling they're doing type assertion in this version if I remember correctly. I think it's better to check if they fix it in later version so that we can bump and fix